src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/infrastructure/clients/DocumentDownloadClient.java

```diff
@@ -1,57 +1,45 @@
 package uk.gov.hmcts.reform.iacasedocumentsapi.infrastructure.clients;
 
-import static java.lang.String.join;
-
-import java.net.MalformedURLException;
-import java.net.URL;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.core.io.Resource;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
-import uk.gov.hmcts.reform.document.DocumentDownloadClientApi;
-import uk.gov.hmcts.reform.iacasedocumentsapi.domain.UserDetailsProvider;
-import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.UserDetails;
+import uk.gov.hmcts.reform.ccd.document.am.feign.CaseDocumentClient;
 import uk.gov.hmcts.reform.iacasedocumentsapi.infrastructure.security.AccessTokenProvider;
 
+@SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
```

Comment:
why do we need to suppress the warning?

Reply:
This has been corrected in the latest commit

Reply:
Was this resolved?